onnx/version_converter/adapters/clip_10_11.h

```diff
@@ -0,0 +1,64 @@
+// Adapter for Clip in default domain from version 10 to 11
+
+#pragma once
+#include <limits>
+
+namespace ONNX_NAMESPACE { namespace version_conversion {
+
+class Clip_10_11 final : public Adapter {
```

Comment:
Why is this adapter needs? After opset 6 clip was only changed in opset 11 so just having a 6 -> 11 adapter should be sufficient right? Is this not the case today?

Reply:
Adapters are always converting between contiguous opset versions. The version converter upgrades operators making jumps of one version point at a time, and if the operatos has changed for a particular jump it goes look for an adapter. So the way Clip is upgraded from 6 to 11 is 6 -> 7 -> 8 -> 9 -> 10 -> 11, where only the last step requires an adapter.

This also answers your comment below.